server/spec/autobind_disabled_for_owner_spec.rb

```diff
@@ -48,17 +48,10 @@
       {'cpu.cpu_socket(s)' => '8'}, nil, owner['key'], [], [])
     consumer_cp = Candlepin.new(nil, nil, consumer.idCert.cert, consumer.idCert['key'])
 
-    begin
-      consumer_cp.consume_product()
-    rescue RestClient::BadRequest => e
-      exception_thrown = true
-      ex_message = "Ignoring request to auto-attach. " +
-        "It is disabled for org \"#{owner['key']}\" because of the content access mode setting."
-      data = JSON.parse(e.response)
-      expect(data['displayMessage']).to eq(ex_message)
-    end
+    consumer_cp.consume_product()
```

Comment:
Let's change the name of this spec test to better reflect the new behaviour. It should say something like `autobind should not attach entitlements when owner is in SCA mode`

Reply:
Updated the spec name.